src/olympia/amo/tests/test_send_mail.py

```diff
@@ -269,6 +269,11 @@ def attach_alternative(*args, **kwargs):
 
     @mock.patch('olympia.amo.tasks.EmailMessage')
     def test_async_will_retry_default(self, backend):
+        # Monkeypatch Celerys ".get()" inside async task error
+        # until https://github.com/celery/celery/issues/4661 (which isn't just
+        # about retries but a general regression) fixed.
```

Comment:
This comment and the clones below should mention that this celery issue manifests itself in eager mode so that we don't have a heart attack when quickly scanning comments in that file :)

Reply:
Fixed in 55fc2e2852